developer-docs-site/docs/concepts/staking.md

```diff
@@ -81,6 +81,16 @@ An owner can designate a voter. This enables the voter to participate in governa
 This document describes staking. See [Governance](governance.md) for how to participate in the Aptos on-chain governance using the owner-voter model.
 :::
 
+## Validation on the Aptos blockchain
+
+The following is a high-level description of how validation works on the Aptos blockchain:
+
+- Throughout the duration of an epoch, the following flow of events occurs several times (thousands of times):
+  - A validator leader is selected by a deterministic formula based on the validator reputation determined by validator's performance and stake. **This selection is not done by voting.**
```

Comment:
we do take into account whether you voted at all or not, but that is a minor signal.